docs/app/content-layouts.md

```diff
@@ -0,0 +1,258 @@
+# Layouts
+
+> The Content Module provides Layouts, which are customized displays for viewing and interacting with Items on a
+> Collection Page. Whether your Collection contains blog posts, project management tasks, geo-coordinates, temporal data
+> or beyond; Layouts display Items in a logical and intuitive way.
+
+[[toc]]
+
+## What's a Layout?
+
+<video autoplay muted loop controls title="Layouts Montage">
+	<source src="" type="video/mp4" />
+</video>
+
+Data models vary drastically in shape and purpose, from eCommerce platforms to IoT fleets and everything in between.
+While your Directus Project data may be stored in Collections _(i.e. SQL data tables)_, this table representation is not
+always the most human-friendly way to interact with information. Layouts provide more human-friendly, intuitive displays
+to view and interact with Items in a Collection.
+
+The following sections detail the Layouts managed and supported by the Directus Team. Keep in mind that Directus is
+open-source, modular and extensible. You are free to create any Layout you need on self-hosted and Enterprise Cloud
+Projects. Additionally, Directus Cloud is always adding more Layouts to its Cloud Exclusive Extensions, so check back
+often to stay up-to-date with the latest and greatest. Additionaly, you can join the community on
+[Discord](https://discord.com/invite/directus) and [GitHub](https://github.com/directus/directus/discussions) to find
+thousands of engineers guiding development of the Directus platform.
+
+## How it Works
+
+<video autoplay muted loop controls title="Adjust Item Layouts">
+	<source src="" type="video/mp4" />
+</video>
+
+To adjust an Item's Layout, follow the steps below.
+
+1. Navigate to the Content Module and select the desired Collection.
+2. In the page Sidebar, click **"Layout Options"**.
+3. Choose the desired Layout you want to use.
+4. Customize the Layout as desired.
+
+### Customization Controls
```

Comment:
Do we want to call Layout controls "customization controls"??

Reply:
@benhaynes @rijkvanzanten I opted to call these "customization controls"
idk if there was another concept we wanted to use instead.